From 2894d541ea1056651d1d1cd0b5dbfb7f6d6fcbf4 Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Thu, 27 Jun 2019 03:05:33 +0200
Subject: [PATCH] modules: update OpenWrt

5e771160b8 kernel: bump 4.14 to 4.14.130
9c6fb1d67a kernel: bump 4.14 to 4.14.129
6c1bef833d kernel: bump 4.9 to 4.9.183
467adaf6c5 OpenWrt v18.06.3: revert to branch defaults
4382d4ce19 OpenWrt v18.06.3: adjust config defaults
97ae9e0ccb fstools: block-mount: fix restart of fstab service
25fc20db8b fstools: update to the latest master branch
fc39d5fc45 fstools: media change detection (eg:sdcard) using kernel polling
b463a13881 hostapd: fix multiple security problems

Compile-tested: ramips-mt7621
Runtime-tested: ramips-mt7621 (D-Link DIR-860L B1)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 314375f2..4168b881 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing luci gluon'
 
 OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-18.06
-OPENWRT_COMMIT=fc1dae5be797f54d45f5a61ae17fe548e108dd0d
+OPENWRT_COMMIT=5e771160b81bfec760a5f7f114cc5dc41ff595e6
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
-- 
2.17.1

